features/cellular/UNITTESTS/at/at_cellulardevice/test_at_cellulardevice.cpp

```diff
@@ -118,10 +118,11 @@ void Test_AT_CellularDevice::test_AT_CellularDevice_close_network()
     ATHandler_stub::ref_count = 0;
 
     CHECK(dev.open_network(&fh1));
+    AT_CellularBase_stub::handler_value = AT_CellularBase_stub::handler_at_constructor_value;
     CHECK(ATHandler_stub::ref_count == 1);
 
     dev.close_network();
-    CHECK(ATHandler_stub::ref_count == 0);
+    CHECK(ATHandler_stub::ref_count == -909);
```

Comment:
what is this magic number? where does it come from ? having macro would help?

Reply:
Added define. Wierd number is to separate item from item with refcount zero and deleted.